### PR TITLE
go-1.21 rc4 update

### PR DESCRIPTION
**Pull request by ajayk**

### CODE REVIEW

File: go-1.21.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package:
   name: go-1.21
-  version: 1.21_rc3
-  epoch: 2
+  version: 1.21_rc4
+  epoch: 3
```
Comment (luhring) [inline go-1.21.yaml:4]: Do we need the epoch bump here if we're bumping the version?

**Review by imjasonh — DISMISSED** [review]
Are wolfibot updates not working here? 🤔

File: go-1.21.yaml (hunk 1 above)
Comment (ajayk) [reply go-1.21.yaml:4]: it makes me think that the `epoch`  should be `0` here

**Comment by ajayk** [comment]
> Are wolfibot updates not working here? 🤔

I Ran the bot but no luck , even the update check fails with no new revision 
@rawlingsj : ^^

**Review by kaniini — APPROVED** [review]

**Pull request merged**